content/en/docs/instrumentation/net/automatic/_index.md

```diff
@@ -0,0 +1,271 @@
+---
+title: Automatic Instrumentation
+linkTitle: Automatic
+weight: 20
+---
+
+This project adds
+[OpenTelemetry instrumentation](https://opentelemetry.io/docs/concepts/instrumenting/#automatic-instrumentation)
+to .NET applications without having to modify their source code.
+
+To see the telemetry from your application directly on the standard output, set
+the following environment variables to `true` before launching your application:
+
+- `OTEL_DOTNET_AUTO_LOGS_CONSOLE_EXPORTER_ENABLED`
+- `OTEL_DOTNET_AUTO_METRICS_CONSOLE_EXPORTER_ENABLED`
+- `OTEL_DOTNET_AUTO_TRACES_CONSOLE_EXPORTER_ENABLED`
+
+For a demo using `docker compose`, clone this repository and follow the
+[examples/demo/README.md](../examples/demo/README.md).
+
+To learn how to manually instrument your service or app code, see
+[Manual instrumentation](../manual).
+
+## Compatibility
+
+OpenTelemetry .NET Automatic Instrumentation should work with all officially
+supported operating systems and versions of
+[.NET](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core).
+
+The minimal supported version of
+[.NET Framework](https://dotnet.microsoft.com/download/dotnet-framework) is
+`4.6.2`.
+
+CI tests run against the following operating systems:
+
+- [Alpine](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/main/docker/alpine.dockerfile)
+- [CentOS 7](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/main/docker/centos-build.dockerfile)
+- [macOS Big Sur 11](https://github.com/actions/runner-images/blob/main/images/macos/macos-11-Readme.md)
+- [Microsoft Windows Server 2022](https://github.com/actions/runner-images/blob/main/images/win/Windows2022-Readme.md)
+- [Ubuntu 20.04 LTS](https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2004-Readme.md)
+
+> **Note** ARM architectures are not supported. See
+> [#2181](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/2181)
+> for more information.
```

Comment:
Make use of the built-in hugo macro (also true for the other instances throughout the document)

```suggestion
{{% alert title="Note" color="warning" %}}
ARM architectures are not supported. See [#2181](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/2181) for more information.
{{% /alert %}}
```

@chalin please double check if this is correct?

Reply:
@svrnm Are these documented somewhere? Or they're just part of Docsy?

Reply:
apologies, they are docsy specific indeed: https://www.docsy.dev/docs/adding-content/shortcodes/#alert